gamemode/modules/util/cl_radial_menu.lua

```diff
@@ -66,6 +102,37 @@ function radialMeta:GetHookIdentifier()
     return "RVR_RadialMenu_" .. tostring( self )
 end
 
+-- From gmod wiki
+local function drawTexturedRectRotatedPoint( x, y, w, h, rot, x0, y0 )
+    local c = math.cos( math.rad( rot ) )
+    local s = math.sin( math.rad( rot ) )
+
+    local newx = y0 * s - x0 * c
+    local newy = y0 * c + x0 * s
+
+    surface.DrawTexturedRectRotated( x + newx, y + newy, w, h, rot )
+end
+
+function radialMeta:DrawOuterCircleSegment( start, size )
+    local centerX, centerY = ScrW() / 2, ScrH() / 2
+    local radius = self.outerRadius * 1.116
```

Comment:
Another option to make configurable? (or at least plop it as a var on `radialMeta`?)

Reply:
Ill pull it out into a var, but its basically just a number calculated using a background material, so is unlikely to ever change